docs/reference/release-notes/highlights-6.4.0.asciidoc

```diff
@@ -7,3 +7,27 @@
 coming[6.4..0]
 
 See also <<release-notes-6.4.0,{es} 6.4.0 release notes>>. 
+
+=== Aggregations
+
+* Auto-interval Date Histogram - A new `auto_date_histogram` aggregaiton has been added which instead of taking an `interval` takes a `buckets` option which defines the maximum number of buckets it should return. The aggregation internally determines the best interval to use to get as close to the `bucket` option as possible without exceeding it. (https://github.com/elastic/elasticsearch/pull/28993)
+
+=== Analysis
+
+* Option to index phrases on text fields - A new `index_phrases` option has been added to `text` fields. When enabled this option will index 2-shingles of the field in a separate Lucene fieldto allow faster, more efficient, phrase searches on that field with the trade-off of consuming a bit more disk space in the index.  (https://github.com/elastic/elasticsearch/pull/30450)
```

Comment:
s/fieldto/field to/

Reply:
I'd say "with the trade-off of consuming more disk space in the index" rather than "a bit more" since we don't try to limit the shingles and they are expensive in terms of disk space.